.github/workflows/dockerbuild.yaml

```
@@ -12,10 +12,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Set GO_VERSION
-        run: echo "GO_VERSION=$(grep -m 1 -r 'go' go.mod | awk '{print $2}')" >> $GITHUB_ENV
-      - uses: actions/setup-go@v3
-        with:
-          go-version: ${{ env.GO_VERSION }}
+      - name: Setup go env
```

The verb for `setup` (none)  is `set up`. 
```suggestion
      - name: Set up Go environment 
```

---

Updated in latest commit